Update webui.sh

ParityError opened the pull request:
Installation should not be assumed to be located within ~/home directory. User should be expected to install project anywhere and run the startup scripts while in stable-diffusion-webui directory.

See issue #8534

ParityError committed:
Update webui-user.sh

Installation should not be assumed to be located within ~/home directory. User should expected to install project anywhere and run the startup scripts while in stable-diffusion-webui directory.

missionfloyd commented:
The idea is that users can edit that file with their own settings.

That's commented out anyway. The default is here.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/dfeee786f903e392dbef1519c7c246b9856ebab3/webui.sh#L26

ParityError commented:
> The idea is that users can edit that file with their own settings.
> 
> That line's commented out anyway, so this change does nothing. The default setting is here.
> 
> https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/dfeee786f903e392dbef1519c7c246b9856ebab3/webui.sh#L26

The `webui.sh` file should not be edited.

```
#!/usr/bin/env bash
#################################################
# Please do not make any changes to this file,  #
# change the variables in webui-user.sh instead #
#################################################
```

ParityError committed:
Update webui.sh

Installation should not be assumed to be located within ~/home directory. User should be expected to install project anywhere and run the startup scripts while in stable-diffusion-webui directory.

See issue #8534

ParityError committed:
Update webui.sh



Installation should not be assumed to be located within ~/home directory. User should be expected to install project anywhere and run the startup scripts while in stable-diffusion-webui directory.

See issue #8534

ParityError committed:
Merge branch 'AUTOMATIC1111:master' into master

AUTOMATIC1111 commented:
1. the readme has some instructions related to webui.sh, please check if they also dont need to be updated
2. webui-user.sh should not be edited unless there really is a need for that, since it has edits by users and will break gill pull for people who don't know how to handle conflicts, and since you're just editing a comment the change definitely should not be there

ParityError commented:
@AUTOMATIC1111, I hadn't thought of that; I will not include webui-user.sh in the pull request.

The only references I could find to webui.sh in the README was this:

> To install in /home/$(whoami)/stable-diffusion-webui/, run:
> 
> bash <(wget -qO- https://raw.githubusercontent.com/AUTOMATIC1111/stable-diffusion-webui/master/webui.sh)

So I would change it to this based on the proposed webui.sh pull request:

> Navigate to the directory you would like the webui to be installed and execute the following command:
> 
> bash <(wget -qO- https://raw.githubusercontent.com/AUTOMATIC1111/stable-diffusion-webui/master/webui.sh)

ParityError committed:
Update webui-user.sh

ParityError committed:
Merge branch 'AUTOMATIC1111:master' into master

ParityError committed:
Update README.md

Updated to reflect change in webui.sh, so that the installation directory is not absolute (/home/user).

ParityError committed:
Merge branch 'AUTOMATIC1111:master' into master

AUTOMATIC1111 committed:
Merge branch 'dev' into master
